cookietemple/config/config.py

```diff
@@ -107,6 +107,14 @@ def config_pat() -> None:
             access_token = cookietemple_questionary_or_dot_cookietemple(
                 "password", "Please enter your Github Access token"
             )
+            # validate input token
+            if not access_token.startswith("ghp_"):  # type: ignore
+                print(
+                    "[bold red]Your personal access token does not seem to have the right format. "
+                    "Please ensure your token starts with [bold green]ghp_"
+                )
+                sys.exit(1)
+            access_token = access_token.strip(" ")  # type: ignore
```

Comment:
@Zethson  I dont think this solves the issue. This must be placed before the check `startswith`!